cypress/e2e/common/cmsSteps.js

```diff
@@ -416,4 +416,102 @@ And('user remembers the source of selected promo image for the mini landing', ()
     cy.get('details img').then($el => {
         imageSrc = $el[0].getAttribute('src')
     })
+});
+
+Then('user selects {string} option from Operations dropdown for content with title {string}', (translateOption, title) => {
+    cy.get(`td:contains('${title}')`).siblings('td').find(`ul.dropbutton >li> a:contains("${translateOption}")`).click({ force: true });
+});
+
+Then('the page title is {string}', (title) => {
+    cy.get(`h2:contains('${title}')`).should('be.visible');
+});
+
+When('user clicks on {string} button to add translation', (addBtn) => {
+    cy.get(`li.add.dropbutton-action a:contains("${addBtn}")`).click({ force: true });
+});
+
+And('Remove button for image was translated as {string}', (removeBtn) => {
+    cy.get(`input[value='${removeBtn}']`).should('be.visible');
+});
+
+And('Related Resources section was translated as {string}', (relatedResources) => {
+    cy.get(`div#field-related-resources-add-more-wrapper h4:contains("${relatedResources}")`).should('be.visible');
+});
+
+And('Link section under related resources was translated as {string}', (linkText) => {
+    cy.get(`summary.seven-details__summary span:contains("${linkText}")`).should('be.visible');
```

Comment:
this might not be a common step based on locator - should prob live under article

Reply:
done